_posts/2016-10-04-5-useful-things-you-wouldn't-think-Siri-can-do-for-you.markdown

```diff
@@ -0,0 +1,80 @@
+
+__Hey Siri, what time is it in Singapore?__
+
+![](/img/2016-10-04-Siri-Time.PNG)
+
+## 4. Messaging
+
+New in iOS 10 are Messenger extensions for Siri. Many people love dictating messages on their phone. Why not take it to the next level and compose the whole message and send it with your favorite messaging app? This lets you keep your hands on the wheel or in the dough and still text with your friends or coworkers using [WhatsApp](https://itunes.apple.com/en/app/whatsapp-messenger/id310633997?mt=8),  [Telegram](https://itunes.apple.com/en/app/telegram-messenger/id686449807?mt=8) or iMessage.
+
+__Hey Siri, text Julia von Kallenberg I'm in a meeting__
+
+![](/img/2016-10-04-Siri-Messages-0.JPG)
+
+To use WhatsApp with this feature I had to go to the Siri Settings and enable WhatsApp in "App Support". After that using it is super convenient and I suspect other messengers, like [Facebook Messenger](https://itunes.apple.com/us/app/messenger/id454638411?mt=8) or [Threema](https://itunes.apple.com/en/app/threema/id578665578?mt=8) to take advantage of that feature in the short term, too.
+
+__Hey Siri, send a text message using WhatsApp__
+
+![](/img/2016-10-04-Siri-Messages-1.JPG)
+
+Just dictate your text and send it off. It will repeat the message again to avoid any awkward moments or typos and you can focus on your work again.
+
+![](/img/2016-10-04-Siri-Messages-2.JPG)
+
+## 5. Photos
+
+Also introduced with iOS 10 is a new feature which lets Photos index your photos using a complicated machine learning algorithm. This will let you search for pictures that contain certain objects like boats, mountains or even sports cars.
+This can be handy if you are putting a presentation together or want to remember that day in the mountains.
+
+__Hey Siri, show me photos of sports cars__
+
+This will open photos with the selected query. You have to unlock your phone to do that but once you do it does a very good job at this.
+
+![](/img/2016-10-04-Siri-Photos.JPG)
+
+## Conclusion
+
+Now that we have seen which five useful things Siri can do for us, we can go on and explore the nearly endless possibilities of Apples personal assistant that speaks your language. If you want to explore more, the folks over at [Hey-Siri.io](http://hey-siri.io "Hey-Siri.io") got you covered. They are currently working on a new version of the website, which takes the brand new Siri extensions of iOS 10 into account. Their website currently supports English and German as well as iOS and macOS Sierra.
+
+Siri can not only help you on your iOS devices but also in your car. If your car supports [CarPlay](http://www.apple.com/ios/carplay/), you can use your integrated Infotainment system to interact with iOS apps using Siri. Helping you to keep your hands on the wheel.
```

Comment:
"hands on the wheel" is there twice in the text. Maybe ok, but a very specific term that sticks in mind, so you feel like you had just read it.